tests/e2e/test_api.py

```diff
@@ -91,11 +91,17 @@ def get_eval_question_answer(qna_pair, qna_id, scenario="without_rag"):
     return eval_query, eval_answer
 
 
+def check_content_type(response, content_type):
+    """Check if response content-type is set to defined value."""
+    return response.headers["content-type"].startswith(content_type)
```

Comment:
return -> assert?

Reply:
ah indeed (I had different logic implemented, then forget to change, good eyes)